dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/bnd.bnd

```diff
@@ -102,12 +108,12 @@ DynamicImport-Package: \
 Private-Package: com.ibm.ws.eclipselink.osgi
 
 Include-Resource: \
-  @${repo;org.eclipse.persistence.antlr;3.5.2.v201711011707;EXACT}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.asm;7.3.1.v202003311108;EXACT}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.core;2.7.7.v20200504-69f2c2b80d;EXACT}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.jpa.jpql;2.7.7.v20200504-69f2c2b80d;EXACT}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.jpa.modelgen;2.7.7.v20200504-69f2c2b80d;EXACT}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.jpa;2.7.7.v20200504-69f2c2b80d;EXACT}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.antlr;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.asm;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.core;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa;${eclFullVersion};EXACT}!/!META-INF/maven/*,\
```

Comment:
The new versions you have in here do not match the old versions.  When I look at the Bundle-Version in the MANIFEST.MF files the old values are right.  It would seem you need to do something similar to what you did in dev/com.ibm.websphere.appserver.thirdparty.eclipselink/bnd.bnd

Reply:
Original: 
`org.eclipse.persistence.core;2.7.7.v20200504-69f2c2b80d

New:
org.eclipse.persistence.core;2.7.7.69f2c2b

Also, the ASM and ANTLR ones are really unique and don't change AS much. Maybe they can be left alone or get their own unique property? But if we are creating a property for exactly one use, we arent gaining much. Not sure which you like more.